concepticondata/conceptlists/Hale-1973-1798.tsv

```diff
@@ -1111,7 +1111,7 @@ Hale-1973-1798-1109	Numbers_and_Quantifiers-51	no one		9.51
 Hale-1973-1798-1110	Numbers_and_Quantifiers-52	another		9.52	197	OTHER
 Hale-1973-1798-1111	Numbers_and_Quantifiers-53	some		9.54	1241	SOME
 Hale-1973-1798-1112	Numbers_and_Quantifiers-54	too much		9.55		
-Hale-1973-1798-1113	Human_Mind_and_Conditions-1	love	n	10a.01		
+Hale-1973-1798-1113	Human_Mind_and_Conditions-1	love	n	10a.01	923	LOVE
```

Comment:
`924 Love` is a verb in concepticon. I'd rather map this to `3834 LOVE (AFFECTION)` https://github.com/concepticon/concepticon-data/blob/master/concepticondata/concepticon.tsv#L3820